pkg/os/launchd/launchd_darwin.go

```diff
@@ -7,6 +7,7 @@ import (
 	goos "os"
 	"os/exec"
 	"path/filepath"
+	"regexp"
 	"strings"
```

Comment:
In the commit log of " Issue #1140 Improve AgentRunning function " can you change "o/p" to "output"? took me a while to expand it :)

Reply:
Done!